components/camera/videosource/webcam.go

```diff
@@ -279,19 +355,10 @@ func NewWebcam(
 	}
 	cam.Monitor()
 
-	s, err := cam.Stream(ctx)
```

Comment:
We added these lines originally for the camera debugger https://github.com/hexbabe/sean-rdk/commit/0a907bf7fdfe4b5649cb0c8598fa41766c542b6f, which is now removed. I think it's confusing and unnecessary to spin up a stream in the constructor anymore since we are not collecting these debug logs. Should just let the error occur when the stream/next spins up naturally